cmd/ancient-store-mem/main.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+func main() {
```

Comment:
Add `doc.go` or `README.md` to document this program. Ie. 

```
	Use:   "ancient-store-mem",
	Short: "Memory-backed remote ancient store application",
	Long: `Uses a memory-backed map to store ancient data.
This application is intended for testing purposed only.
Ancient data is stored ephemerally.
Expects first and only argument to an IPC path, or, the directory
in which a default 'mock-freezer.ipc' path should be created.
Package 'lib' logic may be imported and used in testing contexts as well.
```

Reply:
I will do this.

Reply:
Ah, you did this. https://github.com/etclabscore/core-geth/pull/168/commits/7a7f8d793cbe57d6d99126c7749557029e8551c0